Sources/StreamVideoSwiftUI/Colors.swift

```diff
@@ -19,7 +19,7 @@ public struct Colors {
     public var textInverted = Color(.streamWhite)
     public var onlineIndicatorColor = Color(.streamAccentGreen)
     public var whiteSmoke = Color(.streamWhiteSmoke)
-    public var background: UIColor = .streamWhiteSnow
+    public var background: Color = Color(.streamWhiteSnow)
```

Comment:
I think this will break few of our demo projects. Best to keep it consistent with the chat colors (which are also UIColors). You could write a computed variable though.

Reply:
What's the point of keeping those as UIColors when the rest are Colors?

Reply:
It's a bit of a leftover from the chat SDK, we need to think it through. Best to tackle this in a separate PR.